test_util/test_runner.py

```diff
@@ -50,6 +51,7 @@ def integration_test(
         'PUBLIC_MASTER_HOSTS='+','.join(master_list),
         'SLAVE_HOSTS='+','.join(agent_list),
         'PUBLIC_SLAVE_HOSTS='+','.join(public_agent_list),
+        'DCOS_PLATFORM='+platform,
```

Comment:
Here's where `DCOS_PLATFORM` is being set in the `test_env` from the `platform` passed to `integration_test(...)`.